domains/certificates/Query.API/API/DataSyncSyncer/DataSyncService.cs

```diff
@@ -48,6 +48,12 @@ public DataSyncService(IDataSyncClient client, ILogger<DataSyncService> logger,
                     masterData.MeteringPointOwner,
                     cancellationToken
                 );
+                logger.LogInformation(
+                    "Successfully fetched {numberOfMeasurements} measurements in period from {from} to: {to}",
```

Comment:
Please add GSRN here. That is useful when we have more than 1 metering point.

Reply:
Fixed in e961e28